README.md

```diff
@@ -18,7 +18,41 @@ then it should return the following values after the call:
 [1, 3, 2, 7, 7, 2, 3, 1]
 Notice that it has been doubled in size by having the original sequence appearing in reverse order at the end of the list. You may not make assumptions about how many elements are in the list.
 
+### Anser #1
+
+def mirror(list)
+  n = list.length - 1
+  m = list.length
+  arry = []
+
+  list.each do |l|
+    arry << l
+  end
+
+  m.times do
+    arry << list[n]
+    n -= 1
+  end
+
+  return arry
+end
```

Comment:
This looks good! This problem can be solved with only iterating over the array once, but this solution works too! :)